test/y2storage/storage_manager_test.rb

```diff
@@ -630,7 +630,137 @@
     end
   end
 
+  describe "#staging" do
+    context "when system has not been probed yet" do
```

Comment:
I think that *#staging* and *#probed* tests are repeating the *#probe!* tests. I would keep all the checks about the exceptions in the *#probe!* tests only. And for *#staging* and *#probed* I will simply ensure that `probed!` is used.